merlin.txt

```diff
@@ -0,0 +1 @@
+Make cache backend that django-axes uses configurable through a setting.
```

Comment:
Is this intended to be included in the PR?

Reply:
No it is not, I will remove it.